src/transformers/training_args.py

```diff
@@ -692,6 +692,10 @@ class TrainingArguments:
             for instruction fine-tuning. Check out the [original paper](https://arxiv.org/abs/2310.05914) and the
             [original code](https://github.com/neelsjain/NEFTune). Support transformers `PreTrainedModel` and also
             `PeftModel` from peft.
+
+        batch_eval_metrics (`Optional[bool]`):
+            If set to True, evaluation will call compute_metrics at the end of each batch to accumulate statistics
+            rather than saving all eval logits in memory.
```

Comment:
It would be great to add an example too. It will drive usage ! Do you have an idea where we could potentially put an example @muellerzr ? Otherwise, I think it is worth adding a link to this comment https://github.com/huggingface/transformers/pull/28769/files#r1498479868